---
id:
name:
rule_type: atomic

description: |
  This rule checks ...

accessibility_requirements:
  wcag20:x.x.x: # (Name of Success Criterion)
    forConformance: true | false
    failed: not satisfied
    passed: satisfied | further testing needed
    inapplicable: satisfied | further testing needed

  aria11:x.x.x: # <Heading in WAI-ARIA>
    forConformance: true | false
    failed: not satisfied
    passed: satisfied | further testing needed
    inapplicable: satisfied | further testing needed

  wcag-technique:x.x.x: # <Technique title>
    forConformance: true | false
    failed: not satisfied
    passed: satisfied | further testing needed
    inapplicable: satisfied | further testing needed

input_aspects:
  -  # (e.g. HTTP Messages, DOM Tree, CSS Styling, Accessibility Tree, Language, etc.,)

authors:
  -  # (full names as found in `contributors` property in  `package.json` - if not yet listed, please have authors added to the list)
---

## Applicability ||

Reguła / Zasada ma zasttosowanie do każdego (??) elementu....

## Expectation (1) || Oczekiwania

Każdy element docelowy (badany...) ...

## Assumptions || Założenia

_Nie ma specjalnych założeń_

## Accessibility Support  || Obsługa dostępności

_Nie istnieją żadne istotne problemy związane z dostępnością, znane w odniesieniu do tej zasady.._

## Background  || Tło

- Links to Techniques for WCAG 2.0  || Łacza do technik WCAG
- Latest version: Techniques for WCAG 2.0 W3C Working Group Note 8 April 2014  || Poprzednia wersja
- Link to other methodologies, test specifications or tools || Łącze do innej metodologii, specyfikacji testów lub narzędzi
- Links to test cases, test suites, unit tests, etc. || Linki do przypadków testowych, zestawów testowych, testów jednostkowych itp.
- The WCAG 2.0 Techniques already contain examples and code snippets to illustrate which content passes or fails the test. Whenever possible WCAG-ACT-RULES-CG refers to those. Another source for test cases is the W3C Before and After Demonstration.
  _W WCAG 2.0 Techniki zawierają już przykłady i fragmenty kodu, aby zilustrować, która zawartość przechodzi test, a która nie. W miarę możliwości WCAG-ACT-RULES-CG odnosi się do nich. Innym źródłem dla przypadków testowych jest W3C Before and After Demonstration._
- Other references || Inne zasoby

## Test Cases  || Przypadki testowe

### Passed     || zdane

#### Passed Example 1

Description...

```html
<!-- code -->
```

#### Passed Example 2

...

### Failed     || Niezdane // Awarie

#### Failed Example 1

Description...

```html
<!-- code -->
```

#### Failed Example 2

...

### Inapplicable  || Nieobowiązujące

#### Inapplicable Example 1

Description...

```html
<!-- code -->
```

#### Inapplicable Example 2

...